black = [0,0,0]
#tonos 1
white = [1,1,1]
red = [1,0,0]
green = [0,1,0]
blue = [0,0,1]
yellow = [1,1,0]
cian = [0,1,1]
magenta = [1,0,1]

#tonos 2
white2 = [2,2,2]
red2 = [2,0,0]
green2 = [0,2,0]
blue2 = [0,0,2]
yellow2 = [2,2,0]
cian2 = [0,2,2]
magenta2 = [2,0,2]

#tonos 3
white3 = [3,3,3]
red3 = [3,0,0]
green3 = [0,3,0]
blue3 = [0,0,3]
yellow3 = [3,3,0]
cian3 = [0,3,3]
magenta3 = [3,0,3]
